.gitignore

```
@@ -60,8 +54,12 @@ flatpak-build/
 # Lua
 lua-*
 
-# Clangd 
+# Clangd
 compile_commands.json
 .clangd
 .cache
```

To be consistent with the changes above:
```suggestion
.clangd/
.cache/
```

---

I changed the `.clangd` (which is no longer in use since clangd v11) and `.cache` to append a slash.

With the `.DS_STORE` thing, my changes still match the `.DS_STORE` in any directory - I made a comment about it [here](https://github.com/xournalpp/xournalpp/pull/4551#issuecomment-1368581406).